src/pages/Transaction/TransactionCell/index.tsx

```diff
@@ -138,30 +138,35 @@ const TransactionCellIndexAddress = ({
   )
 }
 
-const isUdt = (cell: State.Cell) => cell.udtInfo && cell.udtInfo.typeHash
+const isUdt = (cell: State.Cell) => {
+  const { extraInfo } = cell
+  const udtExtraInfo = cell.cellType === 'udt' && extraInfo ? (extraInfo as State.UDTInfo) : ''
```

Comment:
The type assertion can be avoided by a type guard

Ref: [typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates)